src/explore-education-statistics-common/src/components/FigureFootnotes.tsx

```diff
@@ -1,18 +1,22 @@
 import CollapsibleList from '@common/components/CollapsibleList';
+import { Footnote } from '@common/services/types/footnotes';
 import React from 'react';
+import VisuallyHidden from './VisuallyHidden';
 
 interface Props {
-  footnotes: {
-    id: string;
-    label: string;
-  }[];
+  footnotes: Footnote[];
+  headingHiddenText?: string;
 }
 
-const FigureFootnotes = ({ footnotes }: Props) => {
+const FigureFootnotes = ({ footnotes, headingHiddenText }: Props) => {
   return footnotes.length > 0 ? (
     <>
-      <h3 className="govuk-heading-m">Footnotes</h3>
-
+      <h3 className="govuk-heading-m">
+        Footnotes
+        {headingHiddenText && (
+          <VisuallyHidden>{`${' '}${headingHiddenText}`}</VisuallyHidden>
```

Comment:
Tiny thing, but you don't need the `${' '}` you can just put a space.

Reply:
I initially thought this too but when I put a space, but it didn't seem to work: 

![image](https://user-images.githubusercontent.com/55030296/190403649-1acfdfd4-d1c4-43e0-a4e5-a8b4c32ce09b.png)

Reply:
That's odd, it seems to work for me when I change it to 
```
<h3 className="govuk-heading-m">
        Footnotes
        {headingHiddenText && (
          <VisuallyHidden>{` ${headingHiddenText}`}</VisuallyHidden>
        )}
      </h3>

Reply:
hmm... I've been having environment issues today so I'll try restarting everything & give it a go again

Reply:
hmm... I've been having environment issues today so I'll try restarting everything & give it a go again

Reply:
Not too sure what was happening with that one. Cleared down my environment + restarted everything & it seems all works well with the changes you suggested